functions/packages.bash

```diff
@@ -24,7 +24,7 @@ samba_setup() {
   echo "OK"
 
   echo -n "$(timestamp) [openHABian] Setting up Samba service... "
-  if ! cond_redirect mkdir -p /var/log/samba /var/run/samba; then echo "FAILED (create directories)"; return 1; fi
```

Comment:
please don't mix multiple issues in one PR

Reply:
Whoops I for got that was in here, I'll remove it.